libs/feature/dataviz/src/lib/chart-view/chart-view.component.html

```diff
@@ -54,4 +57,9 @@
       <span translate>{{ error }}</span>
     </gn-ui-popup-alert>
   </div>
+  <gn-ui-permalink
+    class="mt-2"
+    *ngIf="permalinkBaseUrl"
+    [link]="permalink$ | async"
+  ></gn-ui-permalink>
```

Comment:
I think making the permalink part of this component isn't the best option. This means we'll have to duplicate this for every data preview component. I would say this would be better to have in the metadata view, but maybe there are other issues I'm not thinking of.

Reply:
Good point, I tried to adapt this, but I'm not completely satisfied yet of how the `record-metadata.component` decides whether or not to display the permalink. Maybe this can still be simplified by passing the url to the state (which I finally haven't done yet). What I'm still not seeing really clearly here is the fact that we currently don't have permalinks for table and map. As the three data preview components all exist once `record-metadata.component` is open, I guess that a url state change would have to be triggered upon changing preview tabs (and table/map would have to set the value `null` for now).